### ISSUE TITLE
Result sets

### ISSUE THREAD

**Issue by lily-mayfield** [opensyllabus/osp-api#10]
In regard to returning a result set spanning several requests...

I'm thinking about ditching `limit` and `offset` in favor of `size` and `from` due to this ElasticSearch article: https://www.elastic.co/guide/en/elasticsearch/guide/current/pagination.html

Although here's something about the ElasticSearch `scroll` API which would make next result tokens ideal (possibly): https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html

Wonder if any of you have thoughts on this: @wearpants @davidmcclure @denten 

Note it can be presented however on frontend, but what's happening behind-the-scenes (as mentioned in opening comment) could potentially dramatically affect performance/time.

**Comment by FrancooHM** [comment]
Actually we are using **page** int array for pagination, but we could adapt to any of this options. Actually we really don't have any preferences.

**Comment by lily-mayfield** [comment]
```
{              
 “start”: 1,
 “count”: 20,
 “totalCount”: 100,
 “totalPages”: 5,
 “links”: [
 {
 “href”: “https: //<url>/offset=40&limit=20”,
 “rel”: “next”
 },
 {
 “href”: “https: //<url>/offset=0&limit=20”,
 “rel”: “previous”
 }
 ]
 }
```

something like this but with meta fields (just making some notes)

**Comment by lily-mayfield** [comment]
Okay decided to just go ahead and use `from`/`size` for now! If it becomes an issue we can switch :)

**Comment by lily-mayfield** [comment]
Addressed in #19
